ibft/instance.go

```diff
@@ -117,6 +121,9 @@ func NewInstance(opts InstanceOptions) *Instance {
 // 		broadcast ⟨PRE-PREPARE, λi, ri, inputV aluei⟩ message
 // 		set timer to running and expire after t(ri)
 func (i *Instance) Start(inputValue []byte) {
+	if !i.initialized {
+		i.Logger.Fatal("can't start instance a non  fgs instance")
```

Comment:
what the reason we proceed and not returning in this case?

Reply:
@alonmuroch @nivBlox let's reconsider logger type here
Fatal will crash the node. I don't think faulty of single validator/ibft should affect others

Reply:
@lior-blox @nivBlox we will return error instead